Screens/Search.js

```diff
@@ -34,8 +34,9 @@ const Search = (props) => {
       <Searchbar
         placeholder="Search"
         inputStyle={styles.input}
-        onChangeText={searchQuery => { updateQuery(searchQuery.toLowerCase()) }}
+        onChangeText={searchQuery => updateQuery(searchQuery)}
         value={searchQuery}
+        autoCapitalize='none'
```

Comment:
have it so search icon also updates query. 
Also one more bug: we need 4+ letters to get results